src/main/java/org/sagebionetworks/web/client/widget/ReactComponentSpan.java

```diff
@@ -1,20 +1,50 @@
 package org.sagebionetworks.web.client.widget;
 
-import org.gwtbootstrap3.client.ui.html.Span;
+import org.sagebionetworks.web.client.jsinterop.ReactDOMRoot;
+import org.sagebionetworks.web.client.jsinterop.ReactNode;
+
+import com.google.gwt.user.client.ui.InlineHTML;
 
 /**
  * Automatically unmounts the ReactComponent (if any) inside this div when this container is detached/unloaded.
  */
-public class ReactComponentSpan extends Span {
+public class ReactComponentSpan extends InlineHTML {
```

Comment:
Use InlineHTML insead of gwtboostrap3's Span

Reply:
Would the GWT InlineLabel (basic span) work here?